src/Compilers/CSharp/Test/Emit/PDB/PDBUsingTests.cs

```diff
@@ -525,10 +525,10 @@ class A { void M() {  } }
                 options: TestOptions.DebugDll,
                 references: new[]
                 {
-                    // first aliased reference
-                    compilation1.ToMetadataReference(ImmutableArray.Create("X")),
-                    // second unaliased reference 
-                    compilation1.ToMetadataReference(),
+                // first aliased reference
```

Comment:
Interesting. In that case, this seems like a bug. We might have to investigate to know better.